src/runtime/metatype.cs

```diff
@@ -22,6 +22,15 @@ public static IntPtr Initialize()
             return PyCLRMetaType;
         }
 
+        public static void Release()
+        {
+            if (Runtime.Refcount(PyCLRMetaType) > 1)
+            {
+                SlotsHolder.ReleaseTypeSlots(PyCLRMetaType);
+            }
+            Runtime.XDecref(PyCLRMetaType);
+            PyCLRMetaType = IntPtr.Zero;
```

Comment:
Since you are setting `PyCLRMetaType` to 0, I think it would be a good idea to check for it in the beginning of the method, and throw something like `ObjectDisposedException`